docker/config_template.yaml

```diff
@@ -76,10 +80,14 @@ persistence:
             sql:
                 pluginName: "mysql"
                 databaseName: "{{ default .Env.VISIBILITY_DBNAME "temporal_visibility" }}"
-                connectAddr: "{{ default .Env.MYSQL_SEEDS "" }}:{{ default .Env.DB_PORT "3306" }}"
+                {{- $vis_seeds := default .Env.VISIBILITY_MYSQL_SEEDS .Env.MYSQL_SEEDS -}}
+                {{- $vis_port := default .Env.VISIBILITY_DB_PORT .Env.DB_PORT -}}
+                connectAddr: "{{ default $vis_seeds "" }}:{{ default $vis_port "3306" }}"
                 connectProtocol: "tcp"
-                user: "{{ default .Env.MYSQL_USER "" }}"
-                password: "{{ default .Env.MYSQL_PWD "" }}"
+                {{- $vis_user := default .Env.VISIBILITY_MYSQL_USER .Env.MYSQL_USER -}}
+                {{- $vis_pwd := default .Env.VISIBILITY_MYSQL_PWD .Env.MYSQL_PWD -}}
```

Comment:
Move this one line up to match with Cassandra (define all 4 vars first then use them).